.github/workflows/ci-test.yml

```diff
@@ -311,6 +317,80 @@ jobs:
           $CARGO $CARGOFLAGS test --features='u32' --no-default-features
       - uses: ./.github/actions/build-teardown
 
+  helm-test:
+    runs-on: ubuntu-latest-xl
+    timeout-minutes: 20
+    needs: prepare
+    if: ${{ needs.prepare.outputs.test-helm == 'true' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: ./.github/actions/check-machine
+        with:
+          min-cpu: 3
+          min-ram: 5
+          webhook-gha-hardware: ${{ secrets.WEBHOOK_GHA_HARDWARE }}
+      - name: Check machine details for minikube
```

Comment:
while the below will lead to very readable steps in gha it wont be easy to run locally, you may want a shell script for ths test.    But, you'll have to be very careful to present usable output to caller.   Not strictly necessary to change.